guides/source/caching_with_rails.md

```diff
@@ -512,12 +512,23 @@ class ProductsController < ApplicationController
 end
 ```
 
-### A note on weak ETags
+### Notes on strong and weak ETags
+ETags generated by Rails are weak by default. Weak ETags allow semantically
+equivalent responses to have the same ETags, even if their bodies do not match
+exactly. This is useful when we don't want the page to be regenerated for minor
+changes in response body.
 
```

Comment:
Here we can give an example of weak ETag, for eg. `W/"618bbc92e2d35ea1945008b42799b0e7"`